webdriver-spec.html

```diff
@@ -4898,9 +4884,23 @@ <h3>Find Element</h3>
  </tr>
 </table>
 
-<p>The <dfn>Find Element</dfn> <a>command</a>
- is used to find an <a>element</a> in the <a>current browsing context</a>
- that can be used for future <a>commands</a>.
+<aside class=note>
+<p>
+The <a>Find Element</a> <a>command</a>
+is used to find an <a>element</a> in the <a>current browsing context</a>
+that can be used as the <a>web element</a> context
+for future element-centric <a>commands</a>.
+
+<p>
+For example, consider this pseudo code
+which retrieves an element with the <code>#toremove</code> ID
```

Comment:
This would be better with an html example since we don't say what an ID is.

Reply:
Mind if I file a follow-up to improve command introductions and examples?

Reply:
Filed https://github.com/w3c/webdriver/issues/1233.